### PR TITLE
FIX get_val logistic

### PR DESCRIPTION
**Pull request by QB3**

### CODE REVIEW

File: sparse_ho/tests/test_grid_search.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
+import pytest
 import numpy as np
 from sklearn import linear_model
+from sklearn.model_selection import KFold
+import celer
 from celer.datasets import make_correlated_data
 
 from sparse_ho.utils import Monitor
-from sparse_ho.models import Lasso
+from sparse_ho.models import Lasso  # , ElasticNet, SparseLogreg
```
Comment (agramfort) [inline sparse_ho/tests/test_grid_search.py:9]: cleanup

File: sparse_ho/tests/test_grid_search.py [hunk 2]
```diff
@@ -17,21 +21,82 @@
...
+models["lasso"] = Lasso(estimator=None)
+# models["enet"] = ElasticNet(estimator=None)
+# models["logreg"] = SparseLogreg(estimator=None)
+
+models["lasso_custom"] = Lasso(estimator=celer.Lasso(
+    warm_start=True, fit_intercept=False))
+# custom_models["enet"] = ElasticNet(
+#     estimator=linear_model.ElasticNet(warm_start=True, fit_intercept=False))
+# models["logreg_custom"] = SparseLogreg(
+#     estimator=celer.LogisticRegression(warm_start=True, fit_intercept=False))
```
Comment (agramfort) [inline sparse_ho/tests/test_grid_search.py:53]: we have a lot of code duplication across the tests.

typically things you put in the header of your tests could be factorized in a file `sparse_ho/tests/common.py`

File: sparse_ho/tests/test_criterion.py [hunk 3]
```diff
@@ -1,81 +1,92 @@
...
+    np.geomspace(alpha_max/5, alpha_max/40, num=5))
+dict_list_log_alphas["enet"] = [np.array(i) for i in itertools.product(
+    dict_list_log_alphas["lasso"], dict_list_log_alphas["lasso"])]
+
+list_model_crit = [
+    ('lasso',  HeldOutMSE(idx_train, idx_val)),
+    ('enet', HeldOutMSE(idx_train, idx_val)),
+    ('wLasso', HeldOutMSE(idx_train, idx_val)),
+    ('lasso', FiniteDiffMonteCarloSure(sigma_star)),
+    ('logreg', HeldOutLogistic(idx_train, idx_val))]
```
Comment (agramfort) [inline sparse_ho/tests/test_criterion.py:78]: we have a lot of code duplication across the tests.

typically things you put in the header of your tests could be factorized in a file sparse_ho/tests/common.py

File: sparse_ho/tests/test_grid_search.py (hunk 1 above)
Comment (QB3) [reply sparse_ho/tests/test_grid_search.py]: done :)

File: sparse_ho/tests/test_grid_search.py (hunk 2 above)
Comment (QB3) [reply sparse_ho/tests/test_grid_search.py]: done

File: sparse_ho/tests/test_criterion.py (hunk 3 above)
Comment (QB3) [reply sparse_ho/tests/test_criterion.py:19]: done

Comment (QB3) [reply sparse_ho/tests/test_criterion.py:19]: done

File: sparse_ho/tests/test_grid_search.py (hunk 2 above)
Comment (QB3) [reply sparse_ho/tests/test_grid_search.py]: done

**Comment by agramfort** [comment]
thx @QB3 !

**Pull request merged**